### ISSUE TITLE
Someone take up JSONModel project maintenance? 

### ISSUE THREAD

**Issue by icanzilb** [jsonmodel/jsonmodel#389]
Guys, I've been developing, supporting and resolving issues on JSONModel for about 3 years now
(https://github.com/icanzilb/JSONModel/commit/c80fd1e58f7b9bc7abac25c5550e10374e5e71fa)

As the project becomes more popular it is impossible for me to answer all issues and what is more important there are plenty of pending PRs that are probably merging some quality code but I cannot find the time to look trough and merge them, which is a pity.

Long story short - I need someone to take over and keep the project relevant. 3 years dedicated to this project is more than enough for a single person. I know some of you are keeping an eye on the StackOverflow issues and the GitHub issues and I'd love to give you write access if you'd ask for it.

best, Marin

**Comment by dbachrach** [comment]
Happy to help out.

**Comment by icanzilb** [comment]
Hey Dustin, I would imagine looking trough the pending PRs would be a great starting point. I'm overwhelmed with all the activity but I know there are a number of great PRs waiting to be merged. Still very few people write tests though :(((

I'll give you write access and will help you out if needed just @ ping me in the issue or PR

**Comment by adil-appifytech** [comment]
I use this project on regular basis. I will be more than happy to spend 1 hr daily to reply how to queries and learn more about this project (fix bugs etc).

**Comment by icanzilb** [comment]
@adil-appifytech that's great - there are an overwhelming amount of all kinds of questions being posted on the GitHub repo, if you can answer some of them that's awesome

**Comment by billinghamj** [comment]
It'd be particularly nice if we had someone involved who uses Swift extensively. Some of the issues/PRs I've skipped over because I don't have enough understanding to make a judgement.
